recipe/meta.yaml

```diff
@@ -165,6 +165,8 @@ outputs:
       number: {{ number }}
       string: "{{ build_string }}"
       skip: true  # [cuda_compiler_version not in (undefined, "None", "9.2")]
+      # Skip aarch64 as the build fails with: Unsupported arch flag: -march=.
+      skip: true  # [aarch64]
```

Comment:
AFAICS, the pyarrow feedstock builds aarch64 just fine, although the cuda-PR on `0.16.x` is building on azure instead of drone. In any case, we shouldn't be skipping so easily, IMO.

Can you try the following in `conda-forge.yml`?
```diff
 appveyor:
   secure: {BINSTAR_TOKEN: tumuXLL8PU75WMnRDemRy02ruEq2RpNxeK3dz0MjFssnosPm2v4EFjfNB4PTotA1}
-provider: {linux_aarch64: default, linux_ppc64le: default, win: azure}
+provider: {linux_aarch64: default, linux_ppc64le: azure, win: azure}
 travis:
```

Reply:
@h-vetinari, thanks for the hint! It seems that building on azure is working out better indeed.

Reply:
It didn't work, so now using another workaround.